openwisp_controller/config/base/vpn.py

```diff
@@ -643,6 +647,20 @@ def register_auto_ip_stopper(cls, func):
         if func not in cls._auto_ip_stopper_funcs:
             cls._auto_ip_stopper_funcs.append(func)
 
+    def _get_unique_checks(self, exclude=None, include_meta_constraints=False):
+        if django.VERSION < (4, 1):
+            # TODO: Remove when dropping support for Django 3.2
+            unique_checks, date_checks = super()._get_unique_checks(exclude)
+        else:
+            unique_checks, date_checks = super()._get_unique_checks(
+                exclude, include_meta_constraints
+            )
+        if self.vpn.config.get('vxlan', [{}])[0].get('vni', 0) == 0:
```

Comment:
we'll need to update this to allow VNI to be set to zero (some networking equipment allow this).